### DISCUSSION TITLE
How does Postal persists data?

### DISCUSSION THREAD

**Discussion by peedrofernandes** [postalserver/postal#1994] [Development discussions]
In the **pre-requisites** area of the installation guide, it's shown that a MariaDB container must be runned with the following command:

```
docker run -d \
   --name postal-mariadb \
   -p 127.0.0.1:3306:3306 \
   --restart always \
   -e MARIADB_DATABASE=postal \
   -e MARIADB_ROOT_PASSWORD=postal \
   mariadb
```

As shown, there is no volume specified to persist the data managed by MariaDB inside that container. How does this happen and where, exactly, is all the email data?

**Comment by willpower232** [comment]
Yes that is an omission I will aim to rectify when I have a moment. See also #1909
